src/app/routes/connector-ui/contract-agreement-page/contract-agreement-page/contract-agreement-page.service.ts

```diff
@@ -46,8 +51,16 @@ export class ContractAgreementPageService {
   }
 
   private fetchData(): Observable<Fetched<ContractAgreementPageData>> {
+    const requestBody = this.activeTerminationFilter
```

Comment:
type this before the ternary, please, also this can become `termination status: this.terminationFilter ?? undefined`